templates/pbuilderrc.erb

```diff
@@ -28,7 +28,7 @@ DEBIAN_SUITES=("squeeze" $UNSTABLE_CODENAME $TESTING_CODENAME $STABLE_CODENAME $
 UBUNTU_SUITES=(<%= @ubuntu_suites %>)
 <% else %>
 # List of Ubuntu suites. Update these when needed.
-UBUNTU_SUITES=("lucid" "precise" "trusty" "utopic" "vivid" "wily" "xenial" "yakkety")
+UBUNTU_SUITES=("lucid" "precise" "trusty" "vivid" "wily" "xenial" "yakkety")
```

Comment:
I don't think so, but the issue with utopic was that those files have dropped off the osmirror, we still have lucid so we could build cows http://osmirror.delivery.puppetlabs.net/ubuntu/dists/